applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1342,47 +1260,49 @@ int unicast_client_config_get(struct bt_conn *conn, enum bt_audio_dir dir, uint3
 	}
 
 	if (dir == BT_AUDIO_DIR_SINK) {
-		if (headsets[headset_idx].sink_stream.codec_cfg == NULL) {
+		if (headsets[headset_idx].sink_stream.bap_stream.codec_cfg == NULL) {
 			LOG_ERR("No codec found for the stream");
 
 			return -ENXIO;
 		}
 
 		if (sampling_rate_hz != NULL) {
-			ret = le_audio_freq_hz_get(headsets[headset_idx].sink_stream.codec_cfg,
-						   sampling_rate_hz);
+			ret = le_audio_freq_hz_get(
+				headsets[headset_idx].sink_stream.bap_stream.codec_cfg,
```

Comment:
Consider renaming all the `sink_stream` and `source_stream` to include something with *cap*

Reply:
Renamed to cap_sink_stream and cap_source_stream